src/Evolution/Systems/RadiationTransport/M1Grey/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+# Distributed under the MIT License.
```

Comment:
I would suggest to wait to add the CMake file and cpp files until you actually add code to them.  The CMake file is not needed until the cpp files are compiled